### DISCUSSION TITLE
Expired License - Revalidating

### DISCUSSION THREAD

**Discussion by nilvasales** [aclap-dev/video-downloadhelper#483] [License & Purchase requests]
### What's your issue?

Hi, I formatted my notebook and when I went to activate my lifetime premium license it said it was Expired and Revalidating. How do I reactivate the extension? Thanks.
![image](https://github.com/aclap-dev/video-downloadhelper/assets/81939016/646014b7-6218-420e-914c-e932658136cd)

### The 10 first digits (only 10) of your license number or email address

joao.nivaldo@gmail.com

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
You should have received an email to reset your counter. Maybe in your spam box.
Anyway, I have reset your installation counter, should be OK now.

**Comment by rodamusone** [comment]
Hello, I'm having the same issue of revalidating my license. I've check my email numerous times even the spam folder.
Program was working fine till a couple of days ago then all of a sudden my key was no longer there.
![Screenshot (1)](https://github.com/aclap-dev/video-downloadhelper/assets/169883347/96593d2c-f8be-4f1a-8a10-284db85f6a06)

**Reply by jcfrog** [reply]
Sorry it seems we have a mailer issue, we are on it.
I have reset your counter, should be OK now.

**Reply by rodamusone** [reply]
Awesome! Thank you so much for the quick response JCFROG! Everything works fine now!
